test/integration/resourcemanager/crddeletionprotection/crddeletionprotection_test.go

```diff
@@ -68,20 +69,26 @@ var _ = Describe("Extension CRDs Webhook Handler", func() {
 		}
 		objects = append(objects, crdObjects...)
 
+		By("Get Kubernetes Version")
+		k8sVersion, err := envtest.GetK8SVersion()
+		Expect(err).NotTo(HaveOccurred())
+
 		By("Apply CRDs")
 		applier, err := kubernetes.NewApplierForConfig(restConfig)
 		Expect(err).NotTo(HaveOccurred())
 
 		c, err := client.New(restConfig, client.Options{})
 		Expect(err).NotTo(HaveOccurred())
 
-		crdDeployer, err := crds.NewCRD(c, applier, true, true)
+		extensionCRDDeployer, err := extensionscrds.NewCRD(c, applier, true, true)
 		Expect(err).NotTo(HaveOccurred())
+		Expect(extensionCRDDeployer.Deploy(ctx)).To(Succeed())
 
-		Expect(crdDeployer.Deploy(ctx)).To(Succeed())
+		etcdCRDDeployer, err := etcd.NewCRD(c, k8sVersion)
+		Expect(err).NotTo(HaveOccurred())
+		Expect(etcdCRDDeployer.Deploy(ctx)).To(Succeed())
 
 		manifestReader := kubernetes.NewManifestReader([]byte(strings.Join([]string{
-			etcd.CRD,
 			resourcemanager.CRD,
 		}, "\n---\n")))
```

Comment:
No longer need to join CRDs with `\n---\n` as there is only 1 CRD now:
```suggestion
		manifestReader := kubernetes.NewManifestReader([]byte(resourcemanager.CRD))
```